misk-jooq/README.md

```diff
@@ -0,0 +1,93 @@
+## misk-jooq
+
+Seeks to integrate jooq with misk. Jooq is an open source project located here -
+https://www.jooq.org/
+
+In order to use jooq in your service you need to a few things:
+
+1. Add this module as a dependency to your project.
+1. You need to generate a jooq model from your database. 
+This is best done from running your migrations, and pointing 
+jooq-code-gen to it. There's an example of how to do this in this 
+   module's build.gradle.kts. We are using it to generate a test db model in 
+   order to test classes in this module.  
+   But I'll walk through the steps here anyway:
+   
+a. Add the below lines to your build.gradle.kts
```

Comment:
Would be good to have a bit more documentation here about why we need this gradle block (even if just pointing to jooq's docs), and any parts to replace made more explicit.

Reply:
I added a little more documenation. Hope that helps.